portal/templates/login/_macros/request-new-sign-in-link.njk

```diff
@@ -1,12 +1,8 @@
 {% macro render(params) %}
-  <p class="govuk-!-margin-bottom-6">You have <b>2</b> attempts remaining.</p>
```

Comment:
Some of your changes here are making the tests fail in `portal ui`
https://github.com/UK-Export-Finance/dtfs2/actions/runs/6744341191/job/18334016118?pr=2268

Reply:
I *think* this mightve been because I needed some other commits from #2201. Hopefully fixed now